### ISSUE TITLE
Link-name not being reported for link with title and whitespace content

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#3997]
The following HTML is not reporting a link-name issue, however NVDA does not announce the `title` attribute and the Chrome accessibility view shows that the content of the anchor overrides the `title`.

```html
<a href="#" title="title">&nbsp;</a>
```

<img width="569" alt="screenshot of Chrome devtools Accessibility Tree showing the link name is whitespace" src="https://user-images.githubusercontent.com/2433219/234927033-0e99c556-1229-4846-bdb8-a98c95d9f6b8.png">
